app/src/main/java/com/usepace/android/messagingcenter/clients/connection_client/MessageCenter.java

```diff
@@ -44,6 +44,43 @@ public static void connect(Context context, ConnectionRequest connectionRequest,
         }
     }
 
+
+    /**
+     *
+     * @param context
+     *
+     *
+     */
+    public static void reInitClient(Context context) {
```

Comment:
should not be accessible by app

Reply:
resolved by adding a new class as discussed